src/scss/components/optional/_cookie-warning.scss

```diff
@@ -0,0 +1,76 @@
+@mixin cookieWarning() {
+	#cookieWarning {
+	    background-color: $black;
+	    position: fixed;
+	    bottom: 0;
+	    width: 100%;
+	    z-index: 999;
+	}
+
+	#cookieWarningInner {
+	    width: 940px;
+	    margin: 0 auto;
+	    padding-top: 8px;
+	    padding-bottom: 8px;
+	    overflow: hidden;
+	}
+
+	#cookieWarningInner .message {
+	    width: 940px;
+	}
+
+	@media screen and (max-width: 960px) {
+	    #cookieWarningInner {
+	        width: 620px;
+	    }
+	    #cookieWarningInner .message {
+	        width: 620px;
+	    }
+	}
+
+	@media screen and (max-width: 640px) {
+	    #cookieWarningInner {
+	        width: 300px;
+	    }
+	    #cookieWarningInner .message {
+	        width: 280px;
+	    }
+	}
+
+	#cookieWarning .message {
+	    float: left;
+	}
+
+	#cookieWarning p {
+	    color: $white;
+	    font-size: 11px;
```

Comment:
Should be using the font mappings from the variable file. Also not sure we have 11px defined there.